### PR TITLE
batch of changes to GCZ setup

### PR DESCRIPTION
**Pull request by martindemko**
So far only for QA1. Tested and worked! I would use it for usegalaxy.cz also and run playbook to fix the current state

### CODE REVIEW

File: templates/galaxy/config/tpv_rules_meta.yml.j2 [hunk 1]
```diff
@@ -66,7 +66,7 @@ tools:
           # Only allow the tool to be executed if the user is an admin
           admin_users = app.config.admin_users
           # last line in block must evaluate to a value - which determines whether the TPV if conditional matches or not
-          not user or user.email not in admin_users
+          not user or (user.email not in admin_users and user.email not in ['galaxy@cesnet.cz','galaxy@usegalaxy.cz'])
```
Comment (martenson) [inline templates/galaxy/config/tpv_rules_meta.yml.j2:69]: Why do we need special configured users for running testing tools?

File: host_vars/galaxy-qa1.galaxy.cloud.e-infra.cz/vars.yml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 galaxy_commit_id: release_24.1
 galaxy_build_client: false
 
+miniconda_version: 24.11.1
```
Comment (martenson) [inline host_vars/galaxy-qa1.galaxy.cloud.e-infra.cz/vars.yml:4]: Do we need to differentiate per host, can't we update the miniconda_version in group_vars for all?

File: templates/galaxy/config/tpv_rules_meta.yml.j2 (hunk 1 above)
Comment (martindemko) [reply templates/galaxy/config/tpv_rules_meta.yml.j2:69]: I believe it's better to do testing as a service user and not as one of us because a real user could leave the group or quit the job. Moreover, there are regular tests running on production servers from Nagios and again, I think it's better to have it configured not to run under a specific real person (especially with admin rights, as we discussed recently). What's your suggestion?

File: host_vars/galaxy-qa1.galaxy.cloud.e-infra.cz/vars.yml (hunk 2 above)
Comment (martindemko) [reply host_vars/galaxy-qa1.galaxy.cloud.e-infra.cz/vars.yml:4]: Actually, I wanted to talk about this. I would set this as default because, during testing the infrastructure playbook on two instances, I hit the same trouble with updating Conda. This version works. Though it's not the latest, I didn't want to spend much time with this. If you agree, I would use it as the default.

Comment (martenson) [reply host_vars/galaxy-qa1.galaxy.cloud.e-infra.cz/vars.yml:4]: sounds good, if it works for both 23.0 and 24.2 let's switch the default

File: templates/galaxy/config/tpv_rules_meta.yml.j2 (hunk 1 above)
Comment (martenson) [reply templates/galaxy/config/tpv_rules_meta.yml.j2:69]: I am fine with using service users, but I do not think we need extra tpv rules, if the tools are hidden just let anyone run them maybe?

Comment (martindemko) [reply templates/galaxy/config/tpv_rules_meta.yml.j2:69]: If I need to work on a new tool or use some temporary testing tool to test the boundaries of Galaxy, like a big load on memory or CPU or handling a huge file, I would prefer that no regular user can run it, not even by accident. Although, it's true that we have QA instances, which we didn't back at the time. But those are open to the world anyway, right?

File: host_vars/galaxy-qa1.galaxy.cloud.e-infra.cz/vars.yml (hunk 2 above)
Comment (martindemko) [reply host_vars/galaxy-qa1.galaxy.cloud.e-infra.cz/vars.yml:4]: Ok, I will add it to #73

File: templates/galaxy/config/tpv_rules_meta.yml.j2 (hunk 1 above)
Comment (martenson) [reply templates/galaxy/config/tpv_rules_meta.yml.j2:69]: this rules does not cover any demanding tools though?

```
    cores: 1
    mem: 1
    context:
      walltime: 1
```

**Review by martenson — APPROVED** [review]
